### PR TITLE
symfony 6 and sylius 1.12 support

### PR DESCRIPTION
**Pull request by dsbe-ak**
After releasing new major Version 4 for xynnn/google-tag-manager-bundle, this plugin can now move to symfony 6 / sylius 1.12 as well.
- "sylius/mailer-bundle": "^v2.0.0-BETA.1" is required to force symfony 6

### CODE REVIEW

**Comment by stefandoorn** [comment]
Thanks @dsbe-ak - I've just let CI run.

**Comment by stefandoorn** [comment]
Ah I see what's happening here @dsbe-ak - flex gets installed globally on the CI run, so its not needed to put it in composer.json but the command as suggested in the CI build error should be ran before the require.

**Comment by stefandoorn** [comment]
Getting there :-)

**Comment by stefandoorn** [comment]
Sylius 1.12 also supports Symfony 5 - can we keep it compatible with Symfony 5 + 6? It would make it easier for many to upgrade to Sylius 1.12.

**Comment by dsbe-ak** [comment]
Sylius Standard also uses extra.symfony.require: ^6.0, so I thought it would be a good occation to make a clean cut to symfony 6.
In my current project I'm also moving to symfony 6 with sylius 1.12.

Oh and also I'm not a symfony expert, so I don't know if you can make the test App support 5 and 6 at the same time.

**Comment by stefandoorn** [comment]
The changes that I see should also be compatible with S5, so that would be good.

**Comment by stefandoorn** [comment]
I've changed the permissions a bit, now your Actions will run automatically I think instead me having to approve them first.

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -22,8 +22,8 @@ jobs:
             fail-fast: false
             matrix:
                 php: ["8.1", "8.0"]
-                symfony: ["^5.2"]
-                sylius: ["~1.11.0"]
+                symfony: ["^6.0"]
```
Comment (stefandoorn) [inline .github/workflows/build.yml:25]: What you could try here @dsbe-ak is:

```
symfony: ["^5.2", "^6.0"]
```

That would be great, then we test against both cases :)

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -119,9 +120,9 @@ jobs:
                 name: Validate database schema
                 run: (cd tests/Application && bin/console doctrine:schema:validate)
 
-            -
-                name: Run security check
-                run: symfony security:check
+#            -
```
Comment (stefandoorn) [inline .github/workflows/build.yml:123]: Why does this one need to be turned off?

**Comment by stefandoorn** [comment]
2 last comments - else LGTM.

**Comment by stefandoorn** [comment]
Great - thanks!

**Comment by stefandoorn** [comment]
Released a new version including this as well.

**Pull request merged**